### PR TITLE
Replace repeating code for user and root password in playwright tests

### PR DESCRIPTION
**Pull request by manfredi**
- Description: Replace repeating code for user and root password in playwright tests
- Related ticket: https://progress.opensuse.org/issues/134798
- Verification run:
    + [Overview Devel](https://openqa.opensuse.org/tests/overview?build=issues-134798-user-actor&version=agama-3.0-devel&distri=alp)
    + [Overview Staging](https://openqa.opensuse.org/tests/overview?version=agama-3.0-staging&build=issues-134798-user-actor&distri=alp)

### CODE REVIEW

File: pages/define-user-page.ts [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
     readonly confirmPassword: Locator;
     readonly confirmButton: Locator;
 
     constructor(page: Page) {
         this.page = page;
-        this.userFullName = page.getByLabel('User full name');
-        this.userName = page.getByLabel('Username');
-        this.password = page.getByLabel('User password', { exact: true });
-        this.confirmPassword = page.getByLabel('Password confirmation');
+        this.userFullName = page.locator('#userFullName');
```
Comment (lemon-suse) [inline pages/define-user-page.ts:16]: It is better to sort by alphabetically.

File: tests/full-disk-encryption.spec.ts [hunk 2]
```diff
@@ -27,29 +25,18 @@ test.describe('The main page', () => {
             await storagePage.useEncryption();
 
             const passwordPopup = new EncryptionPasswordPopup(page);
-            await passwordPopup.fillPassword('nots3cr3t');
-            await passwordPopup.fillPasswordConfirmation('nots3cr3t');
+            await passwordPopup.fillPassword(UserActor.user.password);
```
Comment (lemon-suse) [inline tests/full-disk-encryption.spec.ts:28]: Just a minor comments, maybe here a potential risk, just think if we set different password for encryption and user password... Then maybe later PR we can set the interface for storage, and set its own password.

File: pages/define-user-page.ts (hunk 1 above)
Comment (manfredi) [reply pages/define-user-page.ts:16]: Items sorted.

File: tests/full-disk-encryption.spec.ts (hunk 2 above)
Comment (manfredi) [reply tests/full-disk-encryption.spec.ts:28]: We could add new properties to the interface if needs in the future or create a new one that extends the current one. But at the moment it doesn't seem to be useful. Do you prefer to add now these new properties to manage root password and encryption password?

Comment (lemon-suse) [reply tests/full-disk-encryption.spec.ts:28]: I agree with you, we can add that in the future, and now leave it like this is ok for me.

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**